### ISSUE TITLE
[Icon Request]: Support for zod

### ISSUE THREAD

**Issue by selmanozturk** [vscode-icons/vscode-icons#3396]
### Housekeeping

- [X] I'm sure this issue is _not_ a duplicate

### Icon Type

- [X] File Icon
- [ ] Folder Icon

### Icon Type

- [ ] Extension
- [ ] Filename
- [ ] Language ID

### Original Icon

![logo](https://github.com/vscode-icons/vscode-icons/assets/7427741/7ce71ce1-c11b-4f14-8d3c-c51f9860c573)

### Icon Information

Support for the files which contains the definitions of `zod` schemas

### More Information:

`zod` is for typeScript-first schema validation with static type inference. Visit [zod.dev](https://zod.dev/) for more.

**Comment by remcohaszing** [comment]
Correct me if I’m wrong, but as far as I can tell, zod is a TypeScript library, and there’s no language ID for zod. I’m also not aware of any well-known naming best practices that could tell whether a file is related to zod. Please clarify which files should get a zod icon.

**Comment by selmanozturk** [comment]
Correct. There is no file extension related to zod. It was a mistake. I unchecked that option and updated the issue.
In our case, we use a self managed prefix(`*.schemas.ts`) for the separated files those hold zod schemas. We need zod icon to set the icon for those files via `setings.json`.
